tabular/src/autogluon/tabular/models/abstract/abstract_model.py

```diff
@@ -579,15 +583,24 @@ def hyperparameter_tune(self, X_train, y_train, X_val, y_val, scheduler_options,
                 if isinstance(params_copy[hyperparam], Space):
                     logger.log(15, f"{hyperparam}:   {params_copy[hyperparam]}")
 
+        epochs = kwargs.get('epochs', 1)
+
         util_args = dict(
             dataset_train_filename=dataset_train_filename,
             dataset_val_filename=dataset_val_filename,
             directory=directory,
             model=self,
             time_start=time_start,
             time_limit=scheduler_options['time_out'],
+            epochs=epochs,
+            report_probs=kwargs.get('report_probs', False)
```

Comment:
What is report_probs doing?

Reply:
This is a flag for whether validation set probabilities need to be passed to the scheduler. This is needed for the hpo over ensemble.